src/main/java/org/grobid/service/command/TrainingGenerationCommand.java

```diff
@@ -56,30 +65,37 @@ public void configure(Subparser subparser) {
 
     @Override
     protected void run(Bootstrap bootstrap, Namespace namespace, GrobidQuantitiesConfiguration configuration) throws Exception {
+        File grobidHomeOverride = namespace.get(GROBID_HOME_DIRECTORY);
+        String grobidHome = configuration.getGrobidHome();
+
+        initGrobidHome(new File(grobidHome), grobidHomeOverride);
+
+        Path inputDirectory = namespace.get(INPUT_DIRECTORY);
```

Comment:
This gives me an error:
```
java.lang.ClassCastException: java.io.File cannot be cast to java.nio.file.Path
	at org.grobid.service.command.TrainingGenerationCommand.run(TrainingGenerationCommand.java:73)
	at org.grobid.service.command.TrainingGenerationCommand.run(TrainingGenerationCommand.java:22)
	at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:87)
	at io.dropwizard.cli.Cli.run(Cli.java:78)
	at io.dropwizard.Application.run(Application.java:93)
	at org.grobid.service.main.GrobidQuantitiesApplication.main(GrobidQuantitiesApplication.java:76)
```
 
Something like this would fix:
```
File in = namespace.get(INPUT_DIRECTORY);
File out = namespace.get(OUTPUT_DIRECTORY);

Path inputDirectory = in.toPath();
Path outputDirectory = out.toPath();
```

Reply:
I cannot reproduce this error. Wich JVM are you running it with?

Reply:
In my case the parsed args for input and output directory are of type `File`. Using trailing slash in the command or not, doesn't change anything.

```
java -version
openjdk version "1.8.0_282"
OpenJDK Runtime Environment (build 1.8.0_282-8u282-b08-0ubuntu1~20.04-b08)
OpenJDK 64-Bit Server VM (build 25.282-b08, mixed mode)
```